site-src/guides/index.md

```diff
@@ -58,9 +63,46 @@ documentation](/concepts/versioning/).
 To install the experimental channel, run the following kubectl command:
 
 ```bash
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/experimental-install.yaml
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/experimental-install.yaml
 ```
 
+### v1.1 Upgrade Notes
+If you are already using previous versions of GRPCRoute or BackendTLSPolicy
+experimental channel CRDs from previous Gateway API releases, you'll need to be
+careful with this upgrade. If you haven't installed Gateway API before, or have
+exclusively used the standard channel of the API, you can skip the rest of this
+section.
+
+#### GRPCRoute
+**Summary:** If you're already using v1alpha2 GRPCRoute, stick with the
+experimental channel of GRPCRoute in v1.1 until the implementation(s) you're
+relying on have been updated to support GRPCRoute v1.
+
+**Explanation:** With the graduation of GRPCRoute to GA, it is now included in
+standard channel. Unfortunately, that can be problematic for anyone that was
+already using the experimental channel version of GRPCRoute. As a rule, CRDs in
+standard channel do not expose alpha API version to avoid any version
+deprecations in that channel. That means that the standard channel version of
+GRPCRoute excludes v1alpha2. All implementations of GRPCRoute built before the
+v1.1 release of Gateway API would have exclusively relied on v1alpha2 and will
+need to be updated to support GRPCRoute v1. Until implementations have been
+updated to support v1, you can safely upgrade to the experimental channel
+version of GRPCRoute included in v1.1 that exposes both v1 and v1alpha2.
```

Comment:
This basically means that the upgrade path we recommend is to first upgrade the GWAPI CRD to v1.1, and then update the specific implementation, right?

Reply:
Good point, pushed a more detailed set of recommended steps that should make this clearer. PTAL.